README.md

```diff
@@ -1,9 +1,7 @@
 # Use MATLAB with GitLab CI/CD
 You can use [GitLab&trade; CI/CD](https://docs.gitlab.com/ee/ci/index.html) to run MATLAB&reg; scripts, functions, and statements as part of your pipeline. You also can run your MATLAB and Simulink&reg; tests and generate artifacts such as JUnit test results and Cobertura code coverage reports. 
 
-To run MATLAB code and Simulink models with GitLab CI/CD, your runner must have MATLAB (R2019a or later) installed. If you do not have a runner, install [GitLab Runner](https://docs.gitlab.com/runner/install/) and register your computer as a runner for your instance, project, or group.
-
-The runner uses the topmost MATLAB version on the system path. It fails the build if MATLAB is not on the path.  
+To run MATLAB code and Simulink models with GitLab CI/CD, your runner must use the Docker executor to run MATLAB within a container. The [MATLAB Container on Docker Hub](https://www.mathworks.com/help/cloudcenter/ug/matlab-container-on-docker-hub.html) lets you run your build using MATLAB R2020b or a later release. If your build requires additional toolboxes, use a custom MATLAB container instead. For more information on how to create and use a custom MATLAB container, see [Create a Custom MATLAB Container](https://www.mathworks.com/help/cloudcenter/ug/create-a-custom-matlab-container.html).
```

Comment:
This currently gives the impression you must use the Docker executor to run MATLAB in GitLab CI/CD. Can we make it more clear that while this template is geared towards using an Docker executor, you do not have to use one?

Reply:
Hmm. I think the final idea was not to promote a non-Docker-based solution 🤕.

Reply:
I don't think we want to muddy up the template by talking about the non-Docker approach. But I wouldn't say we don't want to promote the non-Docker-based approach. It is perfectly acceptable.

Reply:
In this case, we need to add a section explaining a lot of thing such as:

- MATLAB being on the system path.
- Code snippets on how the jobs must look.

Also, we saw that the last job does not run on Windows in its current form. So we need to provide info on the required OS as well?

@nbhoski, what do you think?

Reply:
I don't think we have to go that far. Just qualifying the "must use a Docker executor" with something like "**in this template**" or something like "While this template shows how to use a Docker executor, it is also possible to use a other executors outside of Docker. Read more about that on https://docs.gitlab.com/runner/executors/".

Reply:
Interesting solution. Maybe this can be a note at the end of the page (in a Notes section)?

Reply:
Added a note in the first section.